src/library/Link/styled.js

```diff
@@ -37,6 +37,11 @@ export const Link = styled('a', {
     // https://developer.mozilla.org/en-US/docs/Web/CSS/:active
     '&:active': {
       color: theme.Link_color_active
-    }
+    },
+
+    /* TargetX Custom Styles */
+    cursor: 'pointer',
+    ...(textDecoration && { textDecoration }),
+    ...(underline && { textDecoration: 'underline' })
```

Comment:
Assuming change above, you don't need this line (L45), just the following is sufficient:
```es6
...(textDecoration && { textDecoration }),
```